docs/source/user/api.rst

```diff
@@ -407,6 +407,67 @@ simple, but can be `enabled through configuration <./settings.html#openwisp-radi
 if identity verification is disabled for a particular org, an empty string
 will be acceptable.
 
+Registering to Multiple Organizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An **HTTP 409** response will be returned if an existing user tries to register
+on a URL of different organization. The response will contain a list of organizations
```

Comment:
of **a** different organization **(because the account already exists)**.
The response will contain a list of organizations with which the user has already registered to the system which may be shown to the user in the UI.